src/Mod/Sketcher/Gui/ViewProviderSketch.cpp

```diff
@@ -3247,7 +3247,7 @@ bool ViewProviderSketch::setEdit(int ModNum)
     if (!sketchDlg)
         sketchDlg = new TaskDlgEditSketch(this);
 
-    connectionToolWidget = sketchDlg->registerToolWidgetChanged(boost::bind(&SketcherGui::ViewProviderSketch::slotToolWidgetChanged, this, bp::_1));
+    connectionToolWidget = sketchDlg->registerToolWidgetChanged(boost::bind(&SketcherGui::ViewProviderSketch::slotToolWidgetChanged, this, boost::placeholders::_1));
```

Comment:
@PaddleStroke 

Using ```boost::placeholders``` was acceptable when we wrote this (actually, I think I did write it). However, in the meantime Werner moved ```boost::bind``` to ```std::bind```. See commit:
68d22d8

Then the good fix here when rebasing, was to move to std::bind.

No need to change anything right now, as I have to revisit that code and I will do it. I just wanted you to know that ```boost::bind``` should not be used and ```std::bind``` should be used instead.

Reply:
Got it.